packaging/linux/build_and_push_packages.sh

```diff
@@ -55,96 +55,64 @@ version="$(cat "$build_dir/VERSION")"
 if [ "$mode" = "devel" ] ; then
   echo "Devel mode does not push. Quitting."
   exit
+elif [ "$mode" != "prerelease" ] ; then
+  echo "Only the 'prerelease' mode is supported now."
+  exit 1
```

Comment:
We'll eventually clean up the naming here (including maybe the prerelease.keybase.io URL), but for now I don't want to bother with tracking down all the callers. Just deleting dead code while preserving current behavior.